modules/common/log/logs-process.nix

```diff
@@ -0,0 +1,77 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  cfg = config.ghaf.logging.client;
+  listenerAddress = "admin-vm-debug";
+  listenerPort = config.ghaf.logging.listener.port;
+  sshCommand = "${pkgs.sshpass}/bin/sshpass -p ghaf ${pkgs.openssh}/bin/ssh -o StrictHostKeyChecking=no ghaf@net-vm";
```

Comment:
Please add a `# TODO replace sshCommand MacCommand with givc rpc` to indicate that the SSH-based integration is not intended as long term

Reply:
Added TODO comment.